### PR TITLE
CB-161: Encounter - On Search By Encounter, "From" date field shows through "Of Type" dropdown

### PR DESCRIPTION
**Pull request by joel-ace**
# JIRA TICKET NAME:
[CB-161: Encounter - On Search By Encounter, "From" date field shows through "Of Type" dropdown](https://issues.openmrs.org/browse/CB-161)

## SUMMARY:
Fix problem of "From" date field showing above "Of Type" dropdown on Search By Encounter form

### CODE REVIEW

File: app/css/cohortbuilder.css [hunk 1]
```diff
@@ -223,4 +223,8 @@ a:hover {
 
 #concept-verbose {
     text-indent: '20px';
-}
\ No newline at end of file
+}
+
+.Select.Select--multi .Select-menu-outer {
```
Comment (andela-dbamidele) [inline app/css/cohortbuilder.css:228]: Is it possible to make the classname lowercase?

And also if you can reduce the dash to one.

Comment (joel-ace) [reply app/css/cohortbuilder.css:228]: I didn't create these classes. I'm just overwriting a rule. I think these classes are from an installed dependency. I'll investigate further

Comment (andela-dbamidele) [reply app/css/cohortbuilder.css:228]: Cool. Nice job 👍

**Comment by jomadoye** [comment]
Looks good.

**Pull request merged**